src/Raven.Server/Documents/Handlers/Admin/AdminLogsHandler.cs

```diff
@@ -164,6 +169,28 @@ await using (var entryStream = entry.Open())
                     }
                 }
 
+                // Add an informational file to the archive if no log files match the specified date range,
+                // ensuring the user receives a non-empty archive with an explanation.
+                using (var archive = new ZipArchive(stream, ZipArchiveMode.Update, true))
```

Comment:
introduce a counter in the prev loop and avoid opening the archive here again if not necessary

Reply:
Done.